tests/_support/AcceptanceTester.php

```diff
@@ -225,4 +225,38 @@ public function deactivateWooCommerce() {
     $I->cli('plugin deactivate woocommerce --allow-root');
   }
 
+  /**
+   * Order a product and create an account within the order process
+   */
+  public function orderProduct(array $product, $user_email, $do_register = true) {
```

Comment:
I think we should split this method into 3 or 4 separate methods. Why? to kame it more futureproof. I am sure someone will come and want to add more arguments like payment method or something else. So we should split it to make it easier. And also we could avoid the boolean argument which is always a good thing™️

Reply:
Will do

Reply:
I agree about splitting because for example I need to put a product to cart and go to checkout, then do test-specific things, only then submit an order.